### PR TITLE
Flag icon updates

### PR DESCRIPTION
**Pull request by Metallicow**
Optimized PNGs
README.rst with table
PNG Extraction code

### CODE REVIEW

File: resources/extracted_flagart_bitmaps/README.rst [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+
+
+Sanity checks
+-------------
+ * https://en.wikipedia.org
+ * http://flagpedia.net/
+
+
++--------------------------+------------+-------------------------------------------+--------------------------------------------------------------------------------------------------+
+| IMG                      | PNG        | Country Name                              | Flag Reference                                                                                   |
```
Comment (RobinD42) [inline resources/extracted_flagart_bitmaps/README.rst]: Please reduce the width of this table to the table readable as text in a window that is 100 characters wide, or at least as close to 100 as you can get it.

**Comment by RobinD42** [comment]
When this PR is finished go ahead and remove the wip label and add the needs-review label. Thanks.

**Comment by Metallicow** [comment]
@RobinD42  It doesn't look like I can edit labels. maybe a repo setting... I'll see if I can finish this tonight...

**Comment by Metallicow** [comment]
This is a very small but helluva feat to accomplish. It probably would be a bit a lot different nowadays..... It may be better for other individuals to conquer with smaller portions. Here is a sample of that works at pulltime. Thanks.

Do it once and do it right 1000 flags, then not much problems singlefold.
Do it the next day after and its like staying 3 nights up with no sleep.
Good luck.

**Pull request closed without merging**